jit-service/src/main/java/org/dcsa/jit/service/notifications/TimestampNotificationMailService.java

```diff
@@ -0,0 +1,100 @@
+package org.dcsa.jit.service.notifications;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.builder.RouteBuilder;
+import org.dcsa.jit.persistence.entity.OperationsEvent;
+import org.dcsa.jit.persistence.entity.OpsEventTimestampDefinition;
+import org.dcsa.jit.persistence.entity.PendingEmailNotification;
+import org.dcsa.jit.persistence.entity.TimestampDefinition;
+import org.dcsa.jit.persistence.repository.OperationsEventRepository;
+import org.dcsa.jit.persistence.repository.OpsEventTimestampDefinitionRepository;
+import org.dcsa.jit.service.notifications.model.FormattedEmail;
+import org.dcsa.jit.service.notifications.model.MailConfiguration;
+import org.dcsa.jit.service.notifications.model.MailTemplate;
+import org.springframework.mail.MailAuthenticationException;
+import org.springframework.mail.MailException;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class TimestampNotificationMailService extends RouteBuilder {
+  private final OperationsEventRepository operationsEventRepository;
+  private final OpsEventTimestampDefinitionRepository opsEventTimestampDefinitionRepository;
+  private final MailConfiguration mailConfiguration;
+  private final EmailFormatter emailFormatter;
+  private final JavaMailSender emailSender;
+
+  @Override
+  public void configure() {
+    from("jpa:org.dcsa.jit.persistence.entity.PendingEmailNotification?namedQuery=PendingEmailNotification.nextPendingEmailNotifications&delay=3600&transacted=true")
+      .bean(this, "processEventMessage");
+  }
+
+  // Called by camel.
+  public void processEventMessage(PendingEmailNotification pendingEmailNotification) {
```

Comment:
any particular reason for usig Spring Mail instead of Camel mail component: https://camel.apache.org/components/3.18.x/mail-component.html

Reply:
Spring mail is way easier. Can add a techdebt task though